packages/bpk-component-radio/src/BpkRadio.scss

```diff
@@ -57,7 +57,7 @@ $bpk-radio-circle-size: $bpk-one-pixel-rem * 10;
 
   &__circle {
     position: absolute;
-    top: ($bpk-spacing-base - $bpk-radio-circle-size) / 2;
+    top: $bpk-spacing-xs;
```

Comment:
I have the same concern as above about intent

Reply:
Using this old value meant that the alignment was off and the box looked all funky this now brings it back to align with the text and the box itself